test-app/app/models/blog.rb

```diff
@@ -6,7 +6,13 @@ class Blog < ActiveRecord::Base
   alias_attribute :headline, :title
   alias_attribute :text, :content
 
-  enum status: [:draft, :published, :archived]
+  # Rails 7.1 deprecated defining enums with keyword arguments.
```

Comment:
Rails 7.1 is not subject to this deprecation

```suggestion
  # Rails 7.2 deprecated defining enums with keyword arguments.
```

However, there are other deprecations related to configuration, but this is not a problem for this PR, it's an existing issue

Ref: https://github.com/talkable/rswag/actions/runs/10330738807/job/28600134064

```
DEPRECATION WARNING: Support for `config.active_support.cache_format_version = 6.1` has been deprecated and will be removed in Rails 7.2.
```